src/elements/content-sidebar/activity-feed/task-new/AssigneeList.js

```diff
@@ -50,47 +48,20 @@ const AvatarDetails = React.memo(({ user, status, completedAt, className }) => {
     );
 });
 
-class AssigneeList extends React.Component<Props, State> {
-    state = {
-        isCollapsed: true,
-    };
-
+class AssigneeList extends React.Component<Props> {
     static defaultProps = {
         initialAssigneeCount: DEFAULT_ASSIGNEES_SHOWN,
         users: {},
     };
 
     listTitleId = uniqueId('assignee-list-title-');
```

Comment:
Is this still being used? If not, this component can be converted to a stateless function component.

Reply:
it is

Reply:
I see it being used in `arial-labelledby={this.listTitleId}`, which has a typo, but I don't see any corresponding label element. Can you link to the line where the label is declared?

Reply:
nvm!